packages/gafl-webapp-service/src/pages/summary/licence-summary/__tests__/route.spec.js

```diff
@@ -244,6 +244,18 @@ describe('licence-summary > route', () => {
 
       expect(licenceTypeDisplay).toHaveBeenCalledWith(mockPermission, catalog)
     })
+
+    it.each([
+      [true, true],
+      [false, false],
```

Comment:
the first element of these two arrays should be string values, not booleans, to mimic the input from an env file. The reason it works is that when you assign values to process.env it casts them to a string.